README.md

```diff
@@ -5,9 +5,10 @@ Currently, it provides widevine CDM support.
 ## Build
 To build the application, you have to clone it first,
 ```
-git clone
+sudo apt install docker docker.io pulseaudio-utils pulseaudio
+git clone https://github.com/monkaBlyat/docker-chromium-armhf
 cd docker-chromium-armhf
-docker build -t hthiemann/chromium-armhf .
+sudo docker build -t hthiemann/chromium-armhf .
```

Comment:
It's an anti-pattern to run docker via sudo. You should add your user to the docker-users group instead

https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user